2024-10-18 19:14:04,677 INFO    Thread-18 (_run_job):8764 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-18 19:14:04,677 INFO    Thread-18 (_run_job):8764 [wandb_setup.py:_flush():79] Configure stats pid to 8764
2024-10-18 19:14:04,677 INFO    Thread-18 (_run_job):8764 [wandb_setup.py:_flush():79] Loading settings from C:\Users\31614\.config\wandb\settings
2024-10-18 19:14:04,677 INFO    Thread-18 (_run_job):8764 [wandb_setup.py:_flush():79] Loading settings from C:\Users\31614\Documents\AI_course_stuff\RLForaging Thesis\RLForagingAgents\RL-models\TD3\wandb\settings
2024-10-18 19:14:04,677 INFO    Thread-18 (_run_job):8764 [wandb_setup.py:_flush():79] Loading settings from environment variables: {'project': 'Pendulum-TD3', 'entity': 'senna-renting-radboud-university', 'root_dir': 'C:\\Users\\31614\\Documents\\AI_course_stuff\\RLForaging Thesis\\RLForagingAgents\\RL-models\\TD3', 'run_id': 'ksyecp36', 'sweep_param_path': 'C:\\Users\\31614\\Documents\\AI_course_stuff\\RLForaging Thesis\\RLForagingAgents\\RL-models\\TD3\\wandb\\sweep-4zxapfjp\\config-ksyecp36.yaml', 'sweep_id': '4zxapfjp'}
2024-10-18 19:14:04,677 INFO    Thread-18 (_run_job):8764 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-18 19:14:04,677 INFO    Thread-18 (_run_job):8764 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-18 19:14:04,677 INFO    Thread-18 (_run_job):8764 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-18 19:14:04,677 INFO    Thread-18 (_run_job):8764 [wandb_init.py:_log_setup():532] Logging user logs to C:\Users\31614\Documents\AI_course_stuff\RLForaging Thesis\RLForagingAgents\RL-models\TD3\wandb\run-20241018_191404-ksyecp36\logs\debug.log
2024-10-18 19:14:04,678 INFO    Thread-18 (_run_job):8764 [wandb_init.py:_log_setup():533] Logging internal logs to C:\Users\31614\Documents\AI_course_stuff\RLForaging Thesis\RLForagingAgents\RL-models\TD3\wandb\run-20241018_191404-ksyecp36\logs\debug-internal.log
2024-10-18 19:14:04,678 INFO    Thread-18 (_run_job):8764 [wandb_init.py:_jupyter_setup():478] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001AA3B43F0E0>
2024-10-18 19:14:04,678 INFO    Thread-18 (_run_job):8764 [wandb_init.py:init():617] calling init triggers
2024-10-18 19:14:04,678 INFO    Thread-18 (_run_job):8764 [wandb_init.py:init():624] wandb.init called with sweep_config: {'batch_size': 256, 'buffer_size': 50000, 'gamma': 0.1362466827310954, 'lr_actor': 0.0059621882698995135, 'lr_critic': 0.0029172171582204543, 'max_length_episode': 200, 'noise_clip': 0.6617412234128096, 'num_episodes': 50, 'policy_interval': 2, 'sigma': 0.4717792776341533, 'sigma_explore': 0.5208428260221403, 'tau': 0.2688397194486837}
config: {}
2024-10-18 19:14:04,678 INFO    Thread-18 (_run_job):8764 [wandb_init.py:init():667] starting backend
2024-10-18 19:14:04,678 INFO    Thread-18 (_run_job):8764 [wandb_init.py:init():671] sending inform_init request
2024-10-18 19:14:04,686 INFO    Thread-18 (_run_job):8764 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn, using: spawn
2024-10-18 19:14:04,686 INFO    Thread-18 (_run_job):8764 [wandb_init.py:init():684] backend started and connected
2024-10-18 19:14:04,703 INFO    Thread-18 (_run_job):8764 [wandb_run.py:_config_callback():1394] config_cb None None {'batch_size': 256, 'buffer_size': 50000, 'gamma': 0.1362466827310954, 'lr_actor': 0.0059621882698995135, 'lr_critic': 0.0029172171582204543, 'max_length_episode': 200, 'noise_clip': 0.6617412234128096, 'num_episodes': 50, 'policy_interval': 2, 'sigma': 0.4717792776341533, 'sigma_explore': 0.5208428260221403, 'tau': 0.2688397194486837}
2024-10-18 19:14:04,703 INFO    Thread-18 (_run_job):8764 [wandb_run.py:_label_probe_notebook():1346] probe notebook
2024-10-18 19:14:04,704 INFO    Thread-18 (_run_job):8764 [wandb_run.py:_label_probe_notebook():1356] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-10-18 19:14:04,704 INFO    Thread-18 (_run_job):8764 [wandb_init.py:init():779] updated telemetry
2024-10-18 19:14:04,736 INFO    Thread-18 (_run_job):8764 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-18 19:14:05,276 INFO    Thread-18 (_run_job):8764 [wandb_init.py:init():863] starting run threads in backend
2024-10-18 19:14:05,791 INFO    Thread-18 (_run_job):8764 [wandb_run.py:_console_start():2465] atexit reg
2024-10-18 19:14:05,792 INFO    Thread-18 (_run_job):8764 [wandb_run.py:_redirect():2313] redirect: wrap_raw
2024-10-18 19:14:05,792 INFO    Thread-18 (_run_job):8764 [wandb_run.py:_redirect():2378] Wrapping output streams.
2024-10-18 19:14:05,792 INFO    Thread-18 (_run_job):8764 [wandb_run.py:_redirect():2403] Redirects installed.
2024-10-18 19:14:05,792 INFO    Thread-18 (_run_job):8764 [wandb_init.py:init():907] run started, returning control to user process
2024-10-18 19:14:19,252 INFO    Thread-18 (_run_job):8764 [wandb_run.py:_finish():2164] finishing run senna-renting-radboud-university/Pendulum-TD3/ksyecp36
2024-10-18 19:14:19,253 INFO    Thread-18 (_run_job):8764 [jupyter.py:save_history():459] not saving jupyter history
2024-10-18 19:14:19,253 INFO    Thread-18 (_run_job):8764 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-18 19:14:19,253 INFO    Thread-18 (_run_job):8764 [wandb_init.py:_jupyter_teardown():460] cleaning up jupyter logic
2024-10-18 19:14:19,253 INFO    Thread-18 (_run_job):8764 [wandb_run.py:_atexit_cleanup():2428] got exitcode: 1
2024-10-18 19:14:19,253 INFO    Thread-18 (_run_job):8764 [wandb_run.py:_restore():2410] restore
2024-10-18 19:14:19,254 INFO    Thread-18 (_run_job):8764 [wandb_run.py:_restore():2416] restore done
2024-10-18 19:14:21,833 INFO    Thread-18 (_run_job):8764 [wandb_run.py:_footer_history_summary_info():4049] rendering history
2024-10-18 19:14:21,833 INFO    Thread-18 (_run_job):8764 [wandb_run.py:_footer_history_summary_info():4081] rendering summary
2024-10-18 19:14:21,839 INFO    Thread-18 (_run_job):8764 [wandb_run.py:_footer_sync_info():4008] logging synced files
